pandas/tests/copy_view/test_astype.py

```diff
@@ -120,14 +120,23 @@ def test_astype_string_copy_on_pickle_roundrip():
     tm.assert_series_equal(base, base_copy)
 
 
+def test_astype_string_copy_on_pickle_roundrip(any_string_dtype):
```

Comment:
i cant find it now, but i thought one of the various string-dtype-like fixtures included `str`, which would let us de-duplicate this test with the previous one

Reply:
The `string_dtype` fixture has that, but that fixture doesn't have all the other variants. After 3.0, the `astype(str)` will be equivalent to `astype(pd.StringDtype(na_value=np.nan))` and then the above test will be superfluous, but at the moment `astype(str)` still does something specfic (do convert to string but end up in object dtype)

Reply:
Will add a comment